changelogs/fragments/57993-openssh_keypair-add-public-key-and-key-comment-validation-and-return-values.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - openssh_keypair - add public key and key comment validation and return values
```

Comment:
It might be better to split this PR up into two parts:
 1) validate public key and comment and update if necessary; this is more like a bugfix and can be backported to 2.8.x;
 2) return comment; this is a new feature and has to wait for 2.9.0.

At least if you want the improved validation in 2.8.x, that is :)

Reply:
Will split.

Reply:
Split done.
The key comment return will be introduced in another PR.